packages/launcher/src/renderer/apps/AppContainer.js

```diff
@@ -41,8 +44,21 @@ export default class AppContainer extends Component<Props> {
       return <View />
     }
 
+    // TODO: Revert to provided path
```

Comment:
Is it possible to do this before merging to master please?
One thing we might be able to do in order to make things simpler for development could be to see the vault when creating or, so we could install some apps and identities. Would that support you use case?

Reply:
OK I think it's fine for now. I'd rather not have temporary code like this in master but it's something I'll change soon in my branch anyways so it's not going to be left for long.
Let's try to get this PR merged first and I'll update my branch accordingly.